### PR TITLE
Remove obsolete comments

### PR DESCRIPTION
**Pull request by pshipton**
Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

File: debugtools/DDR_Autoblob/src/com/ibm/j9ddr/autoblob/GetNativeDirectory.java [hunk 1]
```diff
@@ -29,8 +29,6 @@
  * We build the natives for the 32 bit version of each platform. (So we have a lin_x86 directory that
  * is used for 32 & 64 bit Linux).
  * 
- * Based on Caroline Gough's PlatformFinder utility.
- * 
  * Andrew Hall.
```
Comment (DanHeidinga) [inline debugtools/DDR_Autoblob/src/com/ibm/j9ddr/autoblob/GetNativeDirectory.java]: Can you remove Andrew's name as well?

**Comment by pshipton** [comment]
Updated

**Comment by DanHeidinga** [comment]
Jenkins test sanity plinux

**Comment by pshipton** [comment]
Do not merge, this breaks some DDR internals.

**Comment by pshipton** [comment]
Apparently `Constants from <name>` is a magic DDR command

**Comment by DanHeidinga** [comment]
Is DDR parsing the `/* Constants from ... ` comments in `j9nonbuilder.h`?

**Comment by pshipton** [comment]
> Is DDR parsing the /* Constants from ... comments in j9nonbuilder.h?
Yes. I have restored them and will start a new (internal) build for testing.

**Comment by pshipton** [comment]
@dmitripivkine can you please review the gc parts. I've removed some references to things that no longer exist. In particular I removed `Note: adapted from GC_ClassSuperclassesIterator in Modron/gc_structs` although there still is a gc_structs/ClassSuperclassesIterator.cpp. Maybe you want the comment fixed instead.

**Comment by DanHeidinga** [comment]
Jenkins test sanity plinux

**Pull request merged**